tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyAjax.java

```diff
@@ -1039,6 +1050,34 @@ public CheckCLDR handleCheck(String path, String fullPath, String value, Options
                 final JSONWriter r = newJSONStatusQuick(sm);
                 r.put("locmap", getJSONLocMap(sm));
                 send(r, out);
+            } else if(what.equals(WHAT_ABSTRACT)) {
+                if(xpath == null || xpath.isEmpty()) {
```

Comment:
Since processRequest is already over 700 lines, this block should definitely go in a subroutine